src/core/appinterface.cpp

```diff
@@ -191,6 +192,11 @@ void AppInterface::closeSentry() const
 #endif
 }
 
+void AppInterface::clearProject() const
+{
+  QgsProject::instance()->clear();
```

Comment:
This is not ideal, I think it'd be better to add a QgisMobileApp:clearProject function in which we mProject->clear(), and call that in AppInterface (i.e. mApp->clearProject()).

I know we refer to QgsProject::instance() in multiple places :) but I feel when it's that easy and uncomplicated to do, we should avoid.

Reply:
Otherwise, super happy with this!